packages/scandipwa/src/component/WishlistItem/WishlistItem.container.js

```diff
@@ -158,6 +159,41 @@ export class WishlistItemContainer extends PureComponent {
             );
     }
 
+    recalculatePrice() {
```

Comment:
The naming of this method is not informative. Please explain what it actually does with the name. Why is this recalculation necessary?
Purpose of this method is not clear. Please comment on it within the code.
You also may want to separate it from this class and make an util from it.
Also, calling it on each rerender is very performance-consuming. Consider implementing some kind of workaround, so that this happens only when actually needed.

Reply:
Moved to Util/Wishlist and updated function call